### PR TITLE
Create dockerfile updated and tested

### PR DESCRIPTION
**Pull request by mirfeder**

### CODE REVIEW

**Comment by alvma1945** [comment]
Resolved merge conflicts. Used Miriam's code for event parameter in main.ts file

File: jugglr/container.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+55dc18c4230b9851a754ba39f6973bd86cd770330cb4458b44313bf66e221272
```
Comment (alvma1945) [inline jugglr/container.txt:1]: Making sure this is intentional and not hardcoded in for testing purposes?

Comment (mirfeder) [reply jugglr/container.txt:1]: this is a file that I'm writing the container ID to when it gets returned - one of the async things I couldn't fix outright. It's not an important file but it's not test data exactly.

Comment (mirfeder) [reply jugglr/container.txt:1]: But to clarify, no, this is not anything hardcoded. it's more like a mini data store for lack of a full db to store relevant info for the time being.

**Pull request merged**